fedimint-client/src/lib.rs

```diff
@@ -455,6 +455,7 @@ fn states_add_instance(
     })
 }
 
+#[derive(Debug)]
```

Comment:
Is this necessary in this PR? Seems unrelated

Reply:
necessary because we reference `FederationToClientMap` within `GatewayClientGen` which is debug